src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISHttpServer.cs

```diff
@@ -191,7 +191,7 @@ public IISHttpContext CreateHttpContext(IntPtr pInProcessHandler)
         ~IISHttpServer()
         {
             // If this finalize is invoked, try our best to block all calls into managed.
-            NativeMethods.http_stop_calls_into_managed();
+            NativeMethods.HttpStopCallsIntoManaged();
```

Comment:
This can't throw right?

Reply:
It doesn't return HRESULT and does no validation.